packages/kit/src/vite/build/utils.js

```diff
@@ -95,9 +95,9 @@ export const get_default_config = function ({ config, input, ssr, outDir }) {
 				input,
 				output: {
 					format: 'esm',
-					entryFileNames: ssr ? '[name].js' : '[name]-[hash].js',
-					chunkFileNames: 'chunks/[name]-[hash].js',
-					assetFileNames: 'assets/[name]-[hash][extname]'
+					entryFileNames: ssr ? '[name].js' : 'immutable/[name]-[hash].js',
```

Comment:
will this also put newly added chunks in `immutable`? If so, it wouldn't fix the issue this is attempting to address. I wonder if specifying the `input` name as I was doing before was safer

Reply:
yes, because of the next line